src/mudblazor/.template.config/template.json

```diff
@@ -18,7 +18,7 @@
     "650B3CE7-2E93-4CC4-9F46-466686815EAA",
     "53bc9b9d-9d6a-45d4-8429-2a2761773502"
   ],
-  "identity": "MudBlazorWebApp.CSharp.8.0",
+  "identity": "MudBlazorWebApp.CSharp.10.0",
```

Comment:
shouldn't be 9.0?

Reply:
Thanks, I switched `aspnetcore` to `release\9.0` and resynced this file, as well as the template language files underneath.